app/services/sample_access_validation_service.rb

```diff
@@ -1,12 +1,14 @@
-class BulkDownloadsSampleValidationService
+class SampleAccessValidationService
```

Comment:
Could you add a comment summarizing what this service does to the top of the file?